integration-tests/api.py

```diff
@@ -14,6 +14,15 @@ def register(username, password):
     else:
         return { "error" : res.status_code }
 
+def login(username, password):
+    data = { "username" : username, "password" : password }
+    url = server + "/auth"
+    res = requests.post(url, json = data)
+    if (res.status_code == 200):
+        return res.json()
+    else:
+        return { "error" : res.status_code }
```

Comment:
Isn't it exactly what the `auth` method does ?

Reply:
No, that's a login, not a register. This way you don't have to delete the moderator each time you execute the script

Reply:
https://github.com/heig-PRO-b04/java-backend/wiki/APIv1.Endpoints#post-auth

Reply:
(it's exactly the same as the code defined on lines 26-33)